.gitmodules

```diff
@@ -1022,3 +1019,6 @@
 [submodule "vendor/grammars/zephir-sublime"]
 	path = vendor/grammars/zephir-sublime
 	url = https://github.com/phalcon/zephir-sublime
+[submodule "vendor/grammars/language-fortran"]
```

Comment:
Did you add this manually? There's a script to switch grammars, please see [*Changing the source of a syntax highlighting grammar*](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#changing-the-source-of-a-syntax-highlighting-grammar).

Reply:
I did use the script, but it failed partway through, so I tried to finish off the changes by hand. It seems they're incomplete.

Reply:
```
$ script/add-grammar --replace fortran.tmbundle https://github.com/dparkins/language-fortran
Checking docker is installed and running
$ docker ps
Deregistering: vendor/grammars/fortran.tmbundle
$ git submodule deinit vendor/grammars/fortran.tmbundle
$ git rm -rf vendor/grammars/fortran.tmbundle
$ script/grammar-compiler update -f
Registering new submodule: vendor/grammars/language-fortran
$ git submodule add -f https://github.com/dparkins/language-fortran vendor/grammars/language-fortran
$ script/grammar-compiler add vendor/grammars/language-fortran
Confirming license
$ script/licensed
  > /Library/Ruby/Site/2.3.0/bundler/resolver.rb:287:in `block in verify_gemfile_dependencies_are_found!': Could not find gem 'rake-compiler (~> 0.9)' in any of the gem sources listed in your Gemfile. (Bundler::GemNotFound)
  > 	from /Library/Ruby/Site/2.3.0/bundler/resolver.rb:255:in `each'
  > 	from /Library/Ruby/Site/2.3.0/bundler/resolver.rb:255:in `verify_gemfile_dependencies_are_found!'
  > 	from /Library/Ruby/Site/2.3.0/bundler/resolver.rb:49:in `start'
  > 	from /Library/Ruby/Site/2.3.0/bundler/resolver.rb:22:in `resolve'
  > 	from /Library/Ruby/Site/2.3.0/bundler/definition.rb:258:in `resolve'
  > 	from /Library/Ruby/Site/2.3.0/bundler/definition.rb:170:in `specs'
  > 	from /Library/Ruby/Site/2.3.0/bundler/definition.rb:237:in `specs_for'
  > 	from /Library/Ruby/Site/2.3.0/bundler/definition.rb:226:in `requested_specs'
  > 	from /Library/Ruby/Site/2.3.0/bundler/runtime.rb:108:in `block in definition_method'
  > 	from /Library/Ruby/Site/2.3.0/bundler/runtime.rb:20:in `setup'
  > 	from /Library/Ruby/Site/2.3.0/bundler.rb:107:in `setup'
  > 	from /Library/Ruby/Site/2.3.0/bundler/setup.rb:20:in `<top (required)>'
  > 	from /Library/Ruby/Site/2.3.0/rubygems/core_ext/kernel_require.rb:54:in `require'
  > 	from /Library/Ruby/Site/2.3.0/rubygems/core_ext/kernel_require.rb:54:in `require'
  > 	from script/licensed:3:in `<main>'
Command failed. Aborting.
```
Notably the license was not updated.